### DISCUSSION TITLE
Option to disable double-click to exit fullscreen

### DISCUSSION THREAD

**Discussion by ScottyWired** [sumatrapdfreader/sumatrapdf#4442] [Ideas]
I'm using F11 fullscreen to read an epub. To navigate comfortably, I'm using a drawpad with stylus to drag the page up and down.
This is mostly working well, definitely the best epub reading experience I've ever had. However I do occasionally double-tap the stylus by accident, causing SumatraPDF to exit F11 fullscreen.

I would like an option to disable this behaviour.

**Comment by GitHubRulesOK** [comment]
Should be less likely in latest pre-release as only working via top right corner.
Offhand unsure of active area but less than 10% x10% ?

Currently cant think of a means to over-ride that command other than a dedicated AutoHotKey listening for mouse actions

**Comment by kjk** [comment]
Like @GitHubRulesOK said, it's vastly improved in [pre-release](https://www.sumatrapdfreader.org/prerelease) by only acting in upper-right corner.

So you still can exit full screen with double-tab in upper-right corner but it's much harder to trigger by accident.
